itkwidgets/integrations/__init__.py

```diff
@@ -155,8 +152,10 @@ def _detect_render_type(data, input_type) -> RenderType:
         return RenderType.IMAGE
     elif isinstance(data, itkwasm.PointSet):
         return RenderType.POINT_SET
-    elif isinstance(data, MultiscaleSpatialImage):
-        return RenderType.IMAGE
+    elif HAVE_MULTISCALE_SPATIAL_IMAGE:
+        from multiscale_spatial_image import MultiscaleSpatialImage:
```

Comment:
Need to remove the colon from the import

Reply:
Fixed :-)